src/site/content/en/blog/shape-detection/index.md

```diff
@@ -245,9 +245,9 @@ try {
 
 Purely checking for the existence of the constructors to feature detect the
 Shape Detection API doesn't suffice, as Chrome on Linux and Chrome OS
-currently still expose the detectors, but they are known to not work
-([bug](https://crbug.com/920961)). As a temporary measure, we instead
-recommend a *defensive programming* approach by doing feature detection
+expose the detectors, but they are known to not work
+(which is working as intended according to this [bug](https://crbug.com/920961)).
+Instead, we recommend a *defensive programming* approach by doing feature detection
```

Comment:
I don't think this is as clear as it could be. What do you think of this: 

>Purely checking for the existence of the constructors to feature detect the Shape Detection API doesn't suffice.
The presence of an interface doesn't tell you whether the underlying platform supports the feature. This is [as intended](https://crbug.com/920961). It's why we recommend...

Reply:
Thanks for the suggestion. This is it.